source/codegen/metadata/nidaqmx/__init__.py

```diff
@@ -1,8 +1,8 @@
-from functions import functions
```

Comment:
Note: I'm change all of these to "relative" imports so that they actually import from the module. Previously they were un-qualified absolute/global imports, which relied on adding this folder to `sys.path`. The immediate problem this solves is that the previous implementation wouldn't allow loading 2 modules in the same interpeter because there can only be one module with the absolute name `functions`, etc.

Also note that the new `importlib` code is both more correct and simpler. This is how importing modules is supposed to work and the old code was hacking around a misunderstanding of module imports.

Normally it is preferred to use absolute imports to import from the module. In this case `nidaqmx.functions`, etc. Since these are pure glue code I think it's useful to preserve the consistency/simplicity/paste-ability of the relative imports.